gbpservice/neutron/nsf/configurator/agents/agent_base.py

```diff
@@ -0,0 +1,153 @@
+
+from gbpservice.neutron.nsf.configurator.lib import (
```

Comment:
This file should not be part of this branch. It is already available in configurator-as-a-single-module branch